cmd/vault.go

```diff
@@ -251,21 +251,25 @@ func CreateVault(cmd *cobra.Command, args []string) error {
 		return cmd.Usage()
 	}
 
-	v, _ := DefaultAPIClient.GetVault(args[0])
+	v, err := DefaultAPIClient.GetVault(args[0])
```

Comment:
Må vi ikke skille mellom "permission denied" og "not found" her?

Reply:
> Må vi ikke skille mellom "permission denied" og "not found" her?

Jepp... so much for quick fixes.